spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,6 +32,55 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile5.0-RC1]]
+=== MicroProfile 5.0 Release Candidate 1 (Q32021)
```

Comment:
From MP 4.1 on, we decided to drop the quarter notion in this file. So this Q32021 notion can be removed

Reply:
Done